en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: news_post
+title: "2016 Fukuoka Ruby Award Competition -Entries to be judged by Matz"
```

Comment:
It looks better if there's a space before `Entries` 
"2016 Fukuoka Ruby Award Competition - Entries to be judged by Matz"